### PR TITLE
feat: allow specifying grpc tunnel option explicitly for install media

### PR DESCRIPTION
**Pull request by utkuozdemir**
By default, generated Talos installation media uses `grpc_tunnel` for SideroLink based on the Omni instance configuration, namely via `--siderolink-use-grpc-tunnel` flag.

Allow overriding this setting in `omnictl download` and in Download Installation Media screen on the web.

On the Download Installation Media screen, the default value of the checkbox is based on the instance default.

Closes siderolabs/omni#388.

## Preview

---

### CODE REVIEW

File: client/pkg/omnictl/download.go [hunk 1]
```diff
@@ -58,6 +63,9 @@ func init() {
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.labels, "initial-labels", nil, "Bake initial labels into the generated installation media")
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.extraKernelArgs, "extra-kernel-args", nil, "Add extra kernel args to the generated installation media")
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.extensions, "extensions", nil, "Generate installation media with extensions pre-installed")
+	downloadCmd.Flags().StringVar(&downloadCmdFlags.siderolinkGRPCTunnelMode, "siderolink-grpc-tunnel-mode",
```
Comment (Unix4ever) [inline client/pkg/omnictl/download.go:66]: Maybe just add a bool flag?
I understand the idea behind AUTO mode, but maybe we can make it work like that:
- no flag set: auto mode.
- flag set to false: override mode to false.
- flag set to true: override mode to true.

Something like pointer to bool could work here, but I'm not sure if flags lib supports that.

Comment (Unix4ever) [reply client/pkg/omnictl/download.go:66]: Not having to remember constants for different modes is better I think.

Comment (utkuozdemir) [reply client/pkg/omnictl/download.go]: did the change to work exactly that way

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**